examples/C++/gstreamer/gst-snowboy/gst-plugin/src/snowboy-detect.h

```diff
@@ -0,0 +1,226 @@
+// include/snowboy-detect.h
```

Comment:
If we don't have modifications, can we keep this as a softlink to the original snowboy-detect.h file?